indico/modules/events/editing/client/js/components/TagManager.module.scss

```diff
@@ -0,0 +1,31 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+.tags-container {
+    max-width: 600px !important;
```

Comment:
max-width vs width? Might look weird if there's only a very short tag and then the whole thing shrinks a lot

Reply:
What do you mean? I used `max-width` so that it is a bit more responsive with smaller screens